### PR TITLE
DEV-438: SDK tutorial

### PR DESCRIPTION
**Pull request by smoreface**
SDK tutorial for the docs with short code examples. Please check my code examples and let me know if the ones I've chosen make sense. 

I'll make a note of which ones to mention in links from other docs accordingly, and set that up in a different PR (e.g., a link in the project docs about how to create a project from a python script with the SDK)

### CODE REVIEW

File: docs/source/guide/sdk.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+## Prepare and manage data with filters
+
+You can also use the SDK to control how tasks appear in the data manager to annotators or reviewers. You can create custom filters and ordering for the tasks based on parameters that you specify with the SDK. This lets you have more granular control over which tasks in your dataset get labeled or reviewed, and in which order.
+
+### Prepare unlabeled data with filters
+
+For example, you can create a filter to prepare tasks to be annotated. For example, if you want annotators to focus on tasks in the first 1000 tasks in a dataset that contain the word "possum" in the field "text" in the task data, do the following: 
+```python
+from label_studio_sdk.data_manager import Filters
```
Comment (makseq) [inline docs/source/guide/sdk.md:127]: @smoreface Please, insert the correct code we discussed earlier

File: docs/source/guide/sdk.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        Filters.value("possum")
+    )
+])
+```
+
+### Manage annotations with filters and the Label Studio SDK
+
+For example, to create a filter that displays only tasks with an ID greater than 42 or that were annotated between November 1, 2021, and now, do the following:
+```python
+from label_studio_sdk.data_manager import Filters
```
Comment (makseq) [inline docs/source/guide/sdk.md:155]: from label_studio_sdk.data_manager import Filters, Column, Type, Operator

**Review by makseq — CHANGES_REQUESTED** [review]

File: docs/source/guide/sdk.md (hunk 1 above)
Comment (smoreface) [reply docs/source/guide/sdk.md]: fixed, thanks!

File: docs/source/guide/sdk.md (hunk 2 above)
Comment (smoreface) [reply docs/source/guide/sdk.md]: done thank you!

**Review by makseq — APPROVED** [review]

File: docs/source/guide/sdk.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        Column.data(text),
+        Operator.CONTAINS,
+        Type.String,
+        Filters.value("possum")
+    )
+])
+```
+
+### Manage annotations with filters and the Label Studio SDK
+
```
Comment (makseq) [inline docs/source/guide/sdk.md:152]: This looks like excess duplication...

**Review by makseq — CHANGES_REQUESTED** [review]

File: docs/source/guide/sdk.md (hunk 3 above)
Comment (smoreface) [reply docs/source/guide/sdk.md:152]: I'll update the headers to be clearer

**Review by makseq — APPROVED** [review]

**Pull request merged**